example/powershell/src/Appointment.psm1

```diff
@@ -0,0 +1,55 @@
+<#
+.SYNOPSIS
+    Returns a new Appointment object with the given title
+
+.PARAMETER Title
+    Title of the Appointment
+#>
+function New-Appointment
```

Comment:
Nice work! The coverage is appearing correctly with the `coverage.xml` being picked up by the extension.
![image](https://github.com/user-attachments/assets/d58fb832-dbf7-4327-a4ff-196e57d7d55e)